internal/server/ranking/ranking.go

```diff
@@ -41,7 +41,7 @@ func s(str string) *string {
 // StatisticalVariable, where lower value means higher ranking.
 // Outer key is Import Name, inner key is Rank Key, value is score.
 var StatsRanking = map[string]map[RankKey]int{
-	"CensusPEP": {{MM: s("CensusPEPSurvey")}: 0}, // Population
+	"CensusPEP": {{MM: s("CensusPEPSurvey"), OP: s("P1Y")}: 0}, // Population
```

Comment:
I think we want: (CC @shifucun )

```
"USCensusPEP_Annual_Population": {{MM: s("CensusPEPSurvey"), OP: s("P1Y")}: 0},
"CensusPEP": {{MM: s("CensusPEPSurvey")}: 1},
```

e.g., search for "PEP" in https://autopush.datacommons.org/tools/statvar#Count_Person

Reply:
Thanks, shanth, this is correct!

Reply:
updated